Makefile

```diff
@@ -0,0 +1,20 @@
+dev: 
```

Comment:
can you add commands to README, a little section just to make people aware they can run this. These are runnable  right?

Reply:
Yes but they requires make installed. ( Is present by default in macos and linux but needs to be installed on windows).